adaptive_shield/manifest.json

```diff
@@ -0,0 +1,53 @@
+{
+    "manifest_version": "2.0.0",
+    "app_id": "adaptive-shield",
+    "app_uuid": "",
```

Comment:
@bgoldberg122 Should we leave this blank until publishing?

Reply:
We can set it at any point once the developer platform app and oauth client have been created. All you would need to do is export that developer platform manifest from the UI and use the `published_app_uuid` field from the export here.